db/migrate/20250605101802_create_available_provider_pairings.rb

```diff
@@ -0,0 +1,10 @@
+class CreateAvailableProviderPairings < ActiveRecord::Migration[8.0]
```

Comment:
can we use the name `LeadProviderDeliveryPartnership`, we landed on this in the end as it was clearer in meaning https://github.com/DFE-Digital/register-early-career-teachers-public/pull/498/files#diff-dc044e66140dddc0daf216e146ff14a4a49cb4eacc9329dc242a3fc09548cf98R1

Reply:
Oops I didn't update my diagram. Yeah I'll swap it over.